samples/src/main/java/com/example/bigquery/LoadPartitionedTable.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_load_table_partitioned]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FormatOptions;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.LoadJobConfiguration;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TimePartitioning;
+import java.util.UUID;
+
+public class LoadPartitionedTable {
+
+  public static void runLoadPartitionedTable() throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "MY_DATASET_NAME";
+    String tableName = "MY_TABLE_NAME";
+    String sourceUri = "/path/to/file.csv";
+    loadPartitionedTable(datasetName, tableName, sourceUri);
+  }
+
+  public static void loadPartitionedTable(String datasetName, String tableName, String sourceUri)
+      throws Exception {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      TableId tableId = TableId.of(datasetName, tableName);
+
+      Schema schema =
+          Schema.of(
+              Field.of("name", StandardSQLTypeName.STRING),
+              Field.of("post_abbr", StandardSQLTypeName.STRING),
+              Field.of("date", StandardSQLTypeName.DATE));
+
+      // Configure time partitioning. For full list of options, see:
+      // https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#TimePartitioning
+      TimePartitioning partitioning =
+          TimePartitioning.newBuilder(TimePartitioning.Type.DAY)
+              .setField("date")
+              .setExpirationMs(7776000000L)
```

Comment:
What is the significance of this value? Can you use `java.time` instead?

```java
.setExpirationMs(Duration.of(1, ChronoUnit.DAY).toMillis()); 
```

Reply:
using java.time here instead.